scripts/mkcloud

```diff
@@ -194,6 +194,17 @@ function error_exit
         '
         $scp root@$adminip:/var/log/*tbz $artifacts_dir/
     fi
+
+    # stop chef-client on nodes so the env will not be altered after a failure
+    ssh $sshopts root@$adminip '
```

Comment:
@jdsn $ssh is not used on other places in the code. I would prefer a cleanup afterwards. This is really just a copy of some lines in the same function...